openedx/core/djangoapps/content/block_structure/config/__init__.py

```diff
@@ -25,6 +25,13 @@ def waffle():
     return WaffleSwitchNamespace(name=WAFFLE_NAMESPACE, log_prefix=u'BlockStructure: ')
 
 
+def waffle_switch(name):
+    """
+    Return the waffle switch associated to this namespace.
+    """
+    return WaffleSwitch(waffle(), name, module_name=__name__)
```

Comment:
> Again, can we avoid this? If not, please add a warning as seen in other comments.

:ok_hand: 

>  Also, you'll need to ensure that the class instance tracking knows how to handle multiple instances with the same name.

It doesn't really know how to handle objects with the same name. But since the instance list is actually a WeakSet, temporary instances which are garbage-collected are not kept in the instance list.

Reply:
Another good reason to stop using this pattern.  We should start a document of things to consider during the documentation toggle effort.  I think it would make sense to fix this as toggles are being documented, so it happens in small chunks (like a module at a time).